scripts/src/java/org/oppia/android/scripts/proto/script_exemptions.proto

```diff
@@ -51,3 +51,18 @@ message KdocValidityExemptions {
   // at the correct lexicographical position in the text proto file.
   repeated string exempted_file_path = 1;
 }
+
+// Exemptions for the TODO check.
+message TodoExemptions {
+  // List of all the TODO exemptions.
+  repeated TodoExemption todo_exemption = 1;
+}
+
+// A Todo exemption indicates the usage of a todo keyword which is not meant to accomplish a TODO
```

Comment:
```suggestion
// A Todo exemption indicates the usage of a TODO keyword which is not meant to accomplish a TODO
```

Nit on consistency.

Ditto everywhere else (generally, always refer to TODOs as fully capitalized unless in identifiers).

Reply:
Done.